cmd/clairctl/clair/clair.go

```diff
@@ -0,0 +1,289 @@
+package clair
+
+import (
+	"strconv"
+	"strings"
+  "math"
+	"sort"
+
+	"github.com/coreos/clair/api/v1"
+	"github.com/spf13/viper"
+	"github.com/coreos/clair/cmd/clairctl/xstrings"
+)
+
+var uri string
+var priority string
+var healthPort int
+
+//Report Reporting Config value
+var Report ReportConfig
+
+//VulnerabiliesCounts Total count of vulnerabilities
```

Comment:
I am not sure if @jzelinskie would agree with me on this but I believe that you could simply use a `map[Severity]int` instead of having this struct. This would simplify `CountAllVulnerabilities` and `RelativeCount`.